CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.69.1] - 2023-1-24
+### Changed
+-- yarpcerrors: export logic to get server and client fault type
```

Comment:
nit: drop extra `-`

Reply:
oops somehow I did not see this comment. :(